packages/core/src/elements/Icon/svgs/Checkmark.tsx

```diff
@@ -20,7 +20,7 @@ export const Checkmark = forwardRef<SVGSVGElement, React.SVGProps<SVGSVGElement>
       xmlns={xmlns}
       className={classNames('fe-icon', className)}
     >
-      <path fill='currentColor' d='M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z' />
```

Comment:
why you removed the currentColor

Reply:
To be able to change the color using styles (fill)